bundles/org.openhab.persistence.dynamodb/src/main/resources/OH-INF/config/config.xml

```diff
@@ -107,10 +132,21 @@
 			<advanced>true</advanced>
 		</parameter>
 
+		<parameter name="expireDays" type="integer" required="false" min="1">
+			<label>Expire time for data, in days</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions